app/controllers/users/movies/viewing_parties_controller.rb

```diff
@@ -1,4 +1,40 @@
 class Users::Movies::ViewingPartiesController < ApplicationController
+  before_action :get_user
   def new
+    @facade = MovieFacade.new(params[:movie_id])
+    @users = User.other_users(@user.id)
   end
+
+  def create
+    party = Party.new(party_params)
+    if party.save
+      create_user_parties(params[:invite], party.id)
+      create_host_party(@user.id, party.id)
+      redirect_to user_path(@user)
+    else
+      flash[:error] = 'Please fill out all fields'
+      redirect_to new_user_movie_viewing_party_path(@user, params[:movie_id])
```

Comment:
Also a good catch (go team!) - note for future, using redirect here to force a reload of page and make an API call.